test/integ/deadline_job_attachments/test_job_attachments.py

```diff
@@ -1333,8 +1333,9 @@ def test_upload_bucket_wrong_account(external_bucket: str, job_attachment_test:
 
     # WHEN
     with pytest.raises(
-        AssetSyncError, match=f"Error checking if object exists in bucket '{external_bucket}'"
+        AssetSyncError, match=f"Error uploading binary file in bucket '{external_bucket}'"
```

Comment:
I'm curious about the exception. When I ran this test, I received an "access denied" error. Was the related logic updated in this PR?

Reply:
Yes, I also saw the "access denied" error message. There were no updates related to this in this PR. I found it a bit strange that this integ test was failing... The error message that's currently showing up seems correct though - it's expected that PutObject would be denied if we try to upload a manifest file to a bucket that we don't have access to.